blocks/blockstore/bloom_cache_test.go

```diff
@@ -104,11 +104,11 @@ func TestHasIsBloomCached(t *testing.T) {
 	block := blocks.NewBlock([]byte("newBlock"))
 
 	cachedbs.PutMany([]blocks.Block{block})
-	if cacheFails != 2 {
-		t.Fatalf("expected two datastore hits: %d", cacheFails)
+	if cacheFails != 1 {
```

Comment:
It always weird me out when i see tests changing. Maybe @Kubuxu is more familiar with this and can check it?